model-optimizer/mo/utils/ir_reader/layer_to_class.py

```diff
@@ -233,6 +233,21 @@ def assign_add_output_result(op: Node):
     op.out_port(0).connect(tmp_result.in_port(0))
 
 
+def topk_add_output_result(op: Node):
+    """
+    Function adds necessary output result node for TopK node
+    :param op:
+    :return:
+    """
+    if op.out_port(0).disconnected():
```

Comment:
nit: may be it makes sense to make the function more generic. If not then please, add an assert in the beginning that the node.soft_get('op') == 'TopK'

Reply:
Assert added.